app/templates/orders/new.hbs

```diff
@@ -28,7 +28,7 @@
     </div>
     <div class="row">
       <div class="ten wide column">
-        {{forms/orders/order-form save='save' data=model.order fields=model.form}}
+        {{forms/orders/order-form save='save' data=model.order fields=model.form settings=settings}}
```

Comment:
where is setting defined, how does this work?

Reply:
@CosmicCoder96 It seems to be working, I tried defining `model.setting` over there which was of no good, What would be the ideal way?

Reply:
@kushthedude it cannot work unless setting model is binded. What is the source? If there is none, then it cannot work by definition, verify it.

Reply:
I tried using model.setting to bind it but to my surprise it made the counter fixed to 60 min and when I used the settings everything was working as expected with just a refresh. If its not correct , Can you please some other method?

Reply:
@kushthedude I am not questioning the method, I am asking you where the `settings` object is coming from if it is not defined in the model. Why would `model.setting` ever work if it is not defined in the route model?

Reply:
@CosmicCoder96 Thanks, Made the appropriate changes.